tests/test_flows.py

```diff
@@ -707,30 +707,28 @@ def test_link_metric_query(self):
         query = self.check_cmdu_type_single("link metric query", 0x0005,
                                             env.controller.mac, env.agents[0].mac,
                                             mid)
+        assert query
 
-        if query:
-            query_tlv = self.check_cmdu_has_tlv_single(query, 0x08)
+        query_tlv = self.check_cmdu_has_tlv_single(query, 0x08)
+        assert query_tlv
```

Comment:
I would actually move the assert inside the check_ function. Yes, there are some cases where we actually could continue the test even if the CMDU is missing, but that's not the common case. In the few cases where we do want to continue, we can still add a try block.

And actually, the assertion isn't even needed, because it's implied when you start using query_tlv below.